modules/core/shared/src/main/scala/gem/syntax/TreeSet.scala

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.syntax
+
+import scala.collection.immutable.TreeSet
+
+final class TreeSetCompanionOps(val self: TreeSet.type) extends AnyVal {
+
+  /** Creates a `TreeSet` from a `List`, provided an `Ordering` is available. */
+  def fromList[A: Ordering](lst: List[A]): TreeSet[A] =
+    TreeSet(lst: _*)
```

Comment:
This may not be worth the small gain but I find the `: _*` syntax awkward and didn't want to replicate it everywhere.

Reply:
👍